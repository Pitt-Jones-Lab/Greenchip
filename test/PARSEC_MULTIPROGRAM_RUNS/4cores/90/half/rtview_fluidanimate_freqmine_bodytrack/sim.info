{'benchmarks_revision': ('cat: .gitid: No such file or directory',),
 'benchmarks_root': ('/homes/grads/dek61/SNIPER/benchmarks',),
 'cmdline': ['/homes/grads/dek61/SNIPER/sniper/run-sniper',
             '-n',
             '4',
             '-m',
             'localhost',
             '-d',
             '/scratch/nbp3/PARSEC_MULTIPROGRAM_SHELL/4cores/90/half/rtview_fluidanimate_freqmine_bodytrack',
             '-c',
             '/homes/grads/nbp3/SNIPER_CONFIGS/gainestown_90_half_4.cfg',
             '-s',
             'stop-by-icount:10000000000',
             '--curdir=/scratch/nbp3/PARSEC_MULTIPROGRAM_SHELL/4cores/90/half/rtview_fluidanimate_freqmine_bodytrack',
             '--trace-manual',
             '-g',
             '--traceinput/enabled=true',
             '-g',
             '--traceinput/emulate_syscalls=true',
             '-g',
             '--traceinput/num_apps=4',
             '-g',
             '--traceinput/trace_prefix=/tmp/tmpm72xt4/run_benchmarks'],
 'git_revision': '68418dbe7cab15c65f1f4d300da0c505329beedd',
 'host': 'sb1',
 'pin_version': 'unknown',
 'rusage': (23204.241419000002,
            884.38455299999998,
            201140,
            0,
            0,
            0,
            47762,
            0,
            0,
            0,
            1336,
            0,
            0,
            0,
            72583003,
            6718681),
 'snipercmd': ['bash',
               '-c',
               '/homes/grads/dek61/SNIPER/sniper/lib/sniper -c /homes/grads/dek61/SNIPER/sniper/config/base.cfg --general/total_cores=4 --general/output_dir=/scratch/nbp3/PARSEC_MULTIPROGRAM_SHELL/4cores/90/half/rtview_fluidanimate_freqmine_bodytrack --config=/homes/grads/dek61/SNIPER/sniper/config/nehalem90.cfg --config=/homes/grads/nbp3/SNIPER_CONFIGS/gainestown_90_half_4.cfg --traceinput/enabled=true --traceinput/emulate_syscalls=true --traceinput/num_apps=4 --traceinput/trace_prefix=/tmp/tmpm72xt4/run_benchmarks -g --traceinput/stop_with_first_app=true -g --traceinput/restart_apps=false -g --hooks/numscripts=1 -g --hooks/script0name=/scratch/nbp3/PARSEC_MULTIPROGRAM_SHELL/4cores/90/half/rtview_fluidanimate_freqmine_bodytrack/sim.scripts.py -g --hooks/script0args='],
 't_elapsed': 11477.880753993988,
 't_start': 1464033965.3290031,
 'tracecmds': [],
 'user': 'nbp3',
 'vmem': 1551912960L}
