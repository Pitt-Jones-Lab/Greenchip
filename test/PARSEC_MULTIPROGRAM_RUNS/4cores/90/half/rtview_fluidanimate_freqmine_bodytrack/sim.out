                                   | Core 0      | Core 1      | Core 2      | Core 3     
  Instructions                     |  2860958269 |  2691629834 |  2789945287 |  1656385451
  Cycles                           |  2588275228 |  2588275228 |  2588275228 |  2588275228
  IPC                              |        1.11 |        1.04 |        1.08 |        0.64
  Time (ns)                        |   973035800 |   973035800 |   973035800 |   973035800
  Idle time (ns)                   |       94032 |   219893772 |      387680 |   349471133
  Idle time (%)                    |        0.0% |       22.6% |        0.0% |       35.9%
Branch predictor stats             |             |             |             |            
  num correct                      |   346929205 |   319012009 |   402132606 |   167520813
  num incorrect                    |    30670067 |    21334928 |    28184230 |    21171062
  misprediction rate               |       8.12% |       6.27% |       6.55% |      11.22%
  mpki                             |       10.72 |        7.93 |       10.10 |       12.78
TLB Summary                        |             |             |             |            
  I-TLB                            |             |             |             |            
    num accesses                   |   397900031 |   349188486 |   360390109 |   190885277
    num misses                     |        1033 |         207 |         191 |         352
    miss rate                      |       0.00% |       0.00% |       0.00% |       0.00%
    mpki                           |        0.00 |        0.00 |        0.00 |        0.00
  D-TLB                            |             |             |             |            
    num accesses                   |  1020331420 |   942526713 |  1198546282 |   503930574
    num misses                     |      610269 |     1087631 |      609390 |      445153
    miss rate                      |       0.06% |       0.12% |       0.05% |       0.09%
    mpki                           |        0.21 |        0.40 |        0.22 |        0.27
  L2 TLB                           |             |             |             |            
    num accesses                   |      611302 |     1087838 |      609581 |      445505
    num misses                     |      120529 |      326815 |      139001 |      117072
    miss rate                      |      19.72% |      30.04% |      22.80% |      26.28%
    mpki                           |        0.04 |        0.12 |        0.05 |        0.07
Cache Summary                      |             |             |             |            
  Cache L1-I                       |             |             |             |            
    num cache accesses             |   397900031 |   349188486 |   360390109 |   190885277
    num cache misses               |    12431671 |      842189 |      807224 |      841257
    miss rate                      |       3.12% |       0.24% |       0.22% |       0.44%
    mpki                           |        4.35 |        0.31 |        0.29 |        0.51
  Cache L1-D                       |             |             |             |            
    num cache accesses             |  1020331420 |   942526713 |  1198546282 |   503930574
    num cache misses               |     4526388 |     5834465 |     6205222 |     4635920
    miss rate                      |       0.44% |       0.62% |       0.52% |       0.92%
    mpki                           |        1.58 |        2.17 |        2.22 |        2.80
  Cache L2                         |             |             |             |            
    num cache accesses             |    17308389 |     7149506 |     7720789 |     5807663
    num cache misses               |     2192752 |     2614714 |     2363835 |     1577215
    miss rate                      |      12.67% |      36.57% |      30.62% |      27.16%
    mpki                           |        0.77 |        0.97 |        0.85 |        0.95
DRAM summary                       |             |             |             |            
  num dram accesses                |    13528862 |           0 |           0 |           0
  average dram access latency (ns) |      108.12 |         inf |         inf |         inf
  average dram queueing delay      |      108.12 |        0.00 |        0.00 |        0.00
Coherency Traffic                  |             |             |             |            
  num loads from dram              |     1441068 |     1384920 |     1742379 |     1124295
  num loads from dram cache        |           0 |           0 |           0 |           0
  num loads from remote cache      |           0 |           0 |           0 |           0
